rpm/SOURCES/couchdb-nouveau

```diff
@@ -0,0 +1,15 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+# Java options
+
+JAVA_OPTS="-server -Djava.awt.headless=true -Xmx2g"
```

Comment:
add

```
JAVA_HOME=/etc/alternatives/jre
```
?

Reply:
Added `$JAVA_HOME`, it seems to work well. Thanks!